src/helpers.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+if (function_exists('horizon_stats')) {
+    function horizon_stats()
+    {
+        $dashboard = new \Laravel\Horizon\Http\Controllers\DashboardStatsController();
```

Comment:
While I think this is a nice addition, I strongly believe that the correct way to do this is the other way around: move the current logic from the `index` method to a place where it can be reused in the controller method as well as here.